charts/gardener/controlplane/charts/application/templates/configmap-cluster-idenetity.yaml

```diff
@@ -0,0 +1,7 @@
+kind: ConfigMap
```

Comment:
typo in file name. But isn't this something that should rather be a CLI parameter to the gardener-apiserver, and it deploys it at start-up? What if people deploy g/g without Helm? This cluster identity will be an essential part for Gardener, so you need to have it.

Reply:
I was also thinking to add it as part of the API Server start up, but was not sure if it is its responsibility. Should I add it as a PostStartHook and remove it from the charts?

Reply:
What were your thoughts that made you unsure about the responsibility?

Reply:
Was wondering if the API Server should depend on a flag that it has no use of or has nothing  to do with the API Server as a server, but a Gardener instance as a whole. After that we had the meeting and we imply to provide it as a chart. Anyway, I am going to add it as part of it.

Reply:
Hm, yes, but that's the best way to ensure this identity exists. If it must be added as part of the deployment then it might likely be missing and will lead to complexity and follow-up issues.